app/models/build-info.js

```diff
@@ -0,0 +1,42 @@
+import DS from 'ember-data';
+import Ember from 'ember';
+
+const TIER1 = [
+    ['x86_64-unknown-linux-gnu', 'Linux'],
+    ['x86_64-apple-darwin', 'macOS'],
+    ['x86_64-pc-windows-gnu', 'Windows (GNU)'],
+    ['x86_64-pc-windows-msvc', 'Windows (MSVC)'],
+];
+
+export default DS.Model.extend({
+    version: DS.belongsTo('version', { async: true }),
+    nightly: DS.attr(),
+    ordering: DS.attr(),
+    stable: DS.attr(),
+
+    has_any_info: Ember.computed('ordering', function() {
+        const ordering = this.get('ordering');
+        return (ordering.stable.length + ordering.nightly.length) > 0;
+    }),
+
+    latest_stable: Ember.computed('ordering.stable', function() {
+        const stables = this.get('ordering.stable');
+        return stables[stables.length - 1];
+    }),
+    latest_nightly: Ember.computed('ordering.nightly', function() {
+        const nightlies = this.get('ordering.nightly');
+        return nightlies[nightlies.length - 1];
+    }),
```

Comment:
If we wanted to be fancy, we could make our own macro `last`. Maybe something like this appears to work:

```js
const last = name => (
  Ember.computed(name, function() {
    const items = this.get(name);
    return items[items.length - 1];
  })
);
```

```js
    latest_stable: last('ordering.stable'),
    latest_nightly: last('ordering.nightly'),
```